PayButton/paybutton-server
# Feat/411 1 chronik get balance

Steps to test:

1. run (changes in package.json version of libraries): make reset-dev
2. open in you browser and see balance: http://localhost:3000/api/address/balance/ecash:prezk74rzl4uw22wm6agcteuwztdvp5a9umns6k0d3
3. Change in constants/index.ts the ecash choice to chronik:
`export const BLOCKCHAIN_CLIENTS_CHOSEN: KeyValueT<number> = {
  ecash: BLOCKCHAIN_CLIENTS_OPTIONS.chronik,
  bitcoincash: BLOCKCHAIN_CLIENTS_OPTIONS.grpc
}`
4. open in you browser and see balance (now from chronik client): http://localhost:3000/api/address/balance/ecash:prezk74rzl4uw22wm6agcteuwztdvp5a9umns6k0d3

Other API endpoints will not work with chronik as they are not implemented yet, this will be addressed in future PRs.
